style.css

```diff
@@ -0,0 +1,648 @@
+  font-style: normal;
+  font-weight: 900;
+  font-size: 16px;
+  line-height: 28px;
+  display: flex;
+  align-items: center;
+  letter-spacing: -0.01em;
+  color: #fff;
+  margin-bottom: 0;
+}
+
+.more {
+  display: none;
+}
+
+.imgDiv {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 30%;
+  height: 100%;
+}
+
+.imgDiv img {
+  height: 106.39888000488281px;
+  width: 107px;
+}
+
+.allArtists {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.artistSection {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 32px 12px;
+  gap: 30px;
+}
+
+.titleDiv {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.titleArtistSection {
+  font-family: "Lato", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  display: flex;
+  align-items: center;
+  letter-spacing: -0.05em;
+  color: var(--black);
+}
+
+.borderBottom {
+  width: 50px;
+  border: 2px solid var(--red);
+}
+
+.artistDiv {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 15px;
+}
+
+.descDiv {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: start;
+  width: 70%;
+  height: 85%;
+}
+
+.artistName {
+  font-family: "Lato", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 15px;
+  line-height: 18px;
+  display: flex;
+  align-items: center;
+  letter-spacing: -0.02em;
+  color: var(--black);
+  margin-bottom: 0;
+}
+
+.artistIntro {
+  font-family: "Lato", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 10px;
+  line-height: 15px;
+  display: flex;
+  align-items: center;
+  letter-spacing: -0.02em;
+  color: var(--red);
+  margin-bottom: 0;
+}
+
+.emptyDiv2 {
+  border: 1.5px solid var(--gray);
+  width: 25px;
+}
+
+.artistDesc {
+  font-family: "Lato", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 9px;
+  line-height: 14px;
+  display: flex;
+  align-items: center;
+  letter-spacing: -0.06em;
+  color: var(--black);
+  margin-bottom: 0;
+}
+
+.moreBtn {
+  width: 100%;
+  padding: 10px 0;
+  background-color: #fbf8f8;
+  border: 2px solid var(--gray);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 7px;
+  font-family: "Lato", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 28px;
+  color: var(--black);
+}
+
+.hamburger {
+  font-size: 2rem;
+  padding-left: 25px;
+  cursor: pointer;
+}
+
+.closeBtn {
+  font-size: 2rem;
+  padding-left: 25px;
+  cursor: pointer;
+  color: var(--red);
+}
+
+.closeBtn:hover {
+  transform: scale(1.2);
+  transition: all 0.7s;
+}
+
+.hamburger:hover {
+  transform: scale(1.2);
+  transition: all 0.7s;
+}
+
+.overlay {
+  height: 100vh;
+  width: 100%;
+  position: fixed;
+  background-color: var(--black);
```

Comment:
- [ ] The background color of your Navbar should be transparent or the same color/pattern as the top section background. Kindly make changes


<img width="152" alt="image" src="https://user-images.githubusercontent.com/52934128/225673984-8dbb87c9-5398-4b8a-b1e9-d71cca2f6960.png">